src/app/tests/suites/certification/Test_TC_OPCREDS_3_7.yaml

```
@@ -148,7 +132,8 @@ tests:
                 saveAs: IPK_TH2
 
       # verification: ""
-    - label: "Step 7.1: Read the commissioner node ID from TH2"
+    - label: 
+          "Step 5: Read the commissioner node ID from TH2"
```

```suggestion
          "Step 6.1: Retrieve the commissioner node ID from TH2"
```
Could you please move this down to step 6 so it's clear why this is needed?

---

Fixed: I reorganized the steps by moving Step 5 to Step 6 and Step 6 to Step 6.1 to ensure that the Node ID retrieval happens before the AddNOC command is sent. Placing Step 5 in Step 6.1 caused the test to fail due to the unavailability of the Node ID. By maintaining Step 6 and moving Step 6 to Step 6.1, the Node ID retrieval now precedes the AddNOC command ensuring successful test execution.